core/src/main/java/com/yubico/yubikit/core/smartcard/scp/Scp11KeyParams.java

```diff
@@ -70,4 +70,13 @@ public Scp11KeyParams(KeyRef keyRef, PublicKey pkSdEcka) {
     public KeyRef getKeyRef() {
         return keyRef;
     }
+
+    public List<X509Certificate> getCertificates() {
+        return certificates;
+    }
+
+    @Nullable
+    public KeyRef getOceKeyRef() {
+        return oceKeyRef;
+    }
```

Comment:
Are these only added for testing? Is there anyway we can do this without having to make additions to the public API?

Or do we think they do actually make sense to have even outside of testing?

Reply:
I think we can go without these for now. I removed them and updated the tests.